core/tests/protractor_desktop/topicsAndSkillsDashboard.js

```diff
@@ -167,13 +204,13 @@ describe('Topics and skills dashboard functionality', function() {
     await topicsAndSkillsDashboardPage.navigateToSkillsTab();
     await topicsAndSkillsDashboardPage.filterSkillsByStatus(
       Constants.SKILL_STATUS_UNASSIGNED);
```

Comment:
If I recall correctly, the `filterSkillsByStatus` function doesn't wait for the filtering to complete. I was thinking of waiting for there to be only one skill visible?

Reply:
Hm, I think it is because the filtering is almost instant. I'm not opposed to waiting for filtering to complete but I do not think waiting for only one skill to be visible is the correct way to go about it -- we should have a more definite check to see if the filtering has completed rather than expect the outcome of the filter to be a certain number of results. @Showtim3 do you have any thoughts on this?

Reply:
@kevintab95 I talked with @seanlip, who found that filtering takes several seconds on production. @Showtim3 this might require fixes on the UI side

Reply:
Oh interesting.. I checked this on the test server with network throttling (including "offline" mode) and it seemed to filter quite fast:
![laoding](https://user-images.githubusercontent.com/11008603/94629104-63c8d200-02df-11eb-8713-24f293e08e13.gif)

@seanlip -- On prod, is there any indication that the filtering is in progress?